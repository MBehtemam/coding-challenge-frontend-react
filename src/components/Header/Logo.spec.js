import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Logo from './Logo';

describe('Test suits for Logo Component', () => {
  it('should match with snapshot', () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('should has correct style rule', () => {
    const tree = renderer.create(<Logo />).toJSON();
    expect(tree).toHaveStyleRule('width', '100px');
    expect(tree).toHaveStyleRule('height', '100px');
  });
});
