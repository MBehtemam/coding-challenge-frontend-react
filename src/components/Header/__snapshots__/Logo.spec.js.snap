// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test suits for Logo Component should match with snapshot 1`] = `
.c0 {
  height: 100px;
  width: 100px;
}

<img
  className="c0"
/>
`;
